lib/richmond/logging.rb

```diff
@@ -1,8 +1,13 @@
 require 'logger'
 
 module Richmond
+
   module Logging
 
+    def note message
+      logger.info message
+    end
+
```

Comment:
Another option could be:

```ruby
Logger.info
```

Just add ```self.info(message0``` to the Logger itself, and have it look up the "current" logger.  It's probably a safer place to do this anyway, as I think users will want one logger for everything -- not an individual logger per class.  Or maybe not.

I know many do this, but I don't think modules are a good way to inject dependencies into a class.  In this case, the class isn't "loggable," it's a "logger user."  But since it's using a Logger, why not just use the Logger?  

I don't inject modules a lot, but when I do it's more of a way to mix-in behaviors are that more central to the objects purpose for existing -- not for injecting dependencies and not for drying up code.